### PR TITLE
Fix linux asan

### PR DESCRIPTION
**Pull request by sethalves**
- fix Linux build when HIFI_MEMORY_DEBUGGING=1

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/999/HighFidelity-Beta-PR10783-517bf0af0fba2c4957c461fc58221995d5f3bfdd.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/999/HighFidelity-Beta-PR10783-517bf0af0fba2c4957c461fc58221995d5f3bfdd.exe

File: libraries/gl/CMakeLists.txt [hunk 1]
```diff
@@ -1,5 +1,6 @@
 set(TARGET_NAME gl)
 setup_hifi_library(OpenGL Qml Quick)
+setup_memory_debugger()
```
Comment (jherico) [inline libraries/gl/CMakeLists.txt]: this one should be superfluous, since you do this in the setup_hifi_library call

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/1000/HighFidelity-Beta-PR10783-9ce173f682cf04a8fb3e32bd2f1d65503e639884.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1000/HighFidelity-Beta-PR10783-9ce173f682cf04a8fb3e32bd2f1d65503e639884.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/1026/HighFidelity-Beta-PR10783-d290f03edba3096b65a63c3a9f7993bdc4782bfc.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/1054/HighFidelity-Beta-PR10783-5af07dfb627e5803bf36e295020c7f9bd6120843.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1054/HighFidelity-Beta-PR10783-5af07dfb627e5803bf36e295020c7f9bd6120843.exe

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1058/HighFidelity-Beta-PR10783-5e2aeb96d3b1c5137119b26bd1135595dc7be950.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/1058/HighFidelity-Beta-PR10783-5e2aeb96d3b1c5137119b26bd1135595dc7be950.dmg

**Comment by sethalves** [comment]
to test:
- no functional change

**Comment by CainFoool** [comment]
Testing.

**Comment by CainFoool** [comment]
# Test Passed

No visible changes/crashing etc when teleporting around, ensuring everything loads etc. 

Let me know if there's anything specific you'd like tested.

###### Intel i5 7600K @ 4Ghz, 16GB DDR4 2133Mhz Corsair Vengeance, Asus Strix GTX 1050Ti 3GB, HTC Vive

**Pull request merged**